Artesian/_GMEPublicOffers/GMEPOfferQuery.py

```diff
@@ -13,57 +17,125 @@
 import asyncio
 import itertools
 import urllib
+from __future__ import annotations
 
 class _GMEPOfferQuery:
     __routePrefix = "extract"
-    def __init__(self, client, requestExecutor, partitionStrategy):
+    def __init__(self, client: _ClientsExecutor , requestExecutor: RequestExecutor, partitionStrategy: DefaultPartitionStrategy) -> None: 
+        """ Inits _GME Public Offer Query
+        
+        Args:
```

Comment:
args are not documented. This 'init' is private, so is not required. remove them.